lib/internal/Magento/Framework/App/Config/MetadataConfigTypeProcessor.php

```diff
@@ -130,8 +130,10 @@ private function processScopeData(
             if (isset($metadata['backendModel'])) {
                 /** @var \Magento\Framework\App\Config\Data\ProcessorInterface $processor */
                 $processor = $this->_processorFactory->get($metadata['backendModel']);
-                $value = $processor->processValue($this->_getValue($data, $path));
-                $this->_setValue($data, $path, $value);
+                if (!is_null($processor)) {
```

Comment:
But... according to the factory get method - there is no option to return null. 
It will just throw an exception
https://github.com/magento/magento2/blob/fea21a82cfb8d4f2365f6fb82db30c2811fc96f3/lib/internal/Magento/Framework/App/Config/Data/ProcessorFactory.php#L38-L49

so I think this fix won't work

Reply:
This method has not changed from 2.4.6-p5.  But my test and production systems fail here with the attempt to dereference a null object after the processorFactory->get() call succeeds, and returns NULL.  No exception is thrown.

Is the $this->_pool[<my misconfigured 'backendModel'>] already initialized as NULL?  Thus it skips the $this->_objectManager->create() and throw?

Is the problem upstream where the `_pool[ ]` array is already corrupted.

Reply:
Regardless, would you recommend closing this PR?

Reply:
Upon reflection, ... the thrown exception would have also prevented the `bin/magento setup:upgrad` from completing.  I would have had to rebuild my systems from scratch.

Reply:
From the debugging session, the extremely lengthy App settings would have a ['backend_model'] setting, set to the removed class as a string value.  946 `processFactory->get()`s would succeed, and this, the 947th would return a NULL result and no exception thrown.

I no longer have the setup to test this.  When I added the `!is_null()` check before execution of the returned result, `bin/magento setup:upgrade` stopped crashing and proceeded with the rest of the needed upgrade that included the uninstall.

If I were to speculate, it would be that the failed ['backend_model'] had already been loaded in to the `private _pool[]` array as a null value, thus the return without the call to `->create()` and without the subsequent exception throw.

I spent about a week looking for where the misconfigured ['backend_model'] was added to the App settings but lacked the expertise to find it.  It seemed to load a large chunk of settings, all at once, as if from the database, or from a cache.

Reply:
Besides, I think #38797 deserves more of your expert attention.

Reply:
Since we don't have clear steps to reproduce - I think we can close this task.